client/my-sites/domains/domain-management/email/index.jsx

```diff
@@ -96,7 +103,16 @@ class Email extends React.Component {
 		const { selectedSite, selectedDomainName, translate } = this.props;
 		let emptyContentProps;
 
-		if ( selectedDomainName ) {
+		if ( isGsuiteRestricted() && ! selectedDomainName ) {
+			emptyContentProps = {
+				title: translate( 'Enable powerful email features.' ),
+				line: translate(
+					'To set up email forwarding, and other email ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'To set up email forwarding, G Suite, and other email services for your site, upgrade your site’s web address to a professional custom domain.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).